inference-engine/src/legacy_api/src/convert_function_to_cnn_network.cpp

```diff
@@ -128,7 +128,11 @@ class CNNLayerCreator : public ::ngraph::AttributeVisitor {
 
     void on_adapter(const std::string& name, ::ngraph::ValueAccessor<void*>& adapter) override {
         if (std::string(node->get_type_name()) != "Constant") {
-            THROW_IE_EXCEPTION << "Error: " << node->get_type_name() << " Op cannot use on_adapter(void*).";
+            const auto data_beg = static_cast<char*>(adapter.get_ptr());
+            const auto data_end = std::next(data_beg, adapter.size());
+            std::string ss_str;
```

Comment:
If you want to omit unnecessary copies.
```
params.insert_or_assign(name, std::string{data_beg, data_end});
```
or
```
params[name] = std::string{data_beg, dataend};
```

Reply:
It is better approach, corrected.